spring-javaformat/spring-javaformat-checkstyle/src/main/resources/io/spring/javaformat/checkstyle/spring-checkstyle.xml

```diff
@@ -113,6 +113,7 @@
 		<module name="com.puppycrawl.tools.checkstyle.checks.UpperEllCheck" />
 		<module name="com.puppycrawl.tools.checkstyle.checks.ArrayTypeStyleCheck" />
 		<module name="com.puppycrawl.tools.checkstyle.checks.OuterTypeFilenameCheck" />
+		<module name="UnnecessarySemicolonInEnumeration"/>
```

Comment:
Note for when we merge this: for consistency, the name should be fully-qualified:

```
com.puppycrawl.tools.checkstyle.checks.coding.UnnecessarySemicolonInEnumerationCheck
```

Reply:
@wilkinsona Thanks! I updated it as you suggested.